sheets/utils.py

```diff
@@ -38,15 +41,181 @@ def generate_google_client_config():
 )
 coupon_assign_sheet_spec = SpreadsheetSpec(
     first_data_row=2,
-    last_data_column="B",
-    num_columns=2,
-    column_headers=["Coupon Code", "Email (Assignee)"],
+    last_data_column="D",
+    num_columns=4,
+    column_headers=["Coupon Code", "Email (Assignee)", "Status", "Status Date"],
+)
+ASSIGNMENT_SHEET_STATUS_COLUMN = 2
```

Comment:
Should this be a .env/settings variable or is it unlikely to ever change?

Reply:
It's sort of a helper constant, and it derives directly from the column headers in the definition above. It's unlikely to change, and if it does, this variable will need to change along with the sheet spec defined immediately above